core/metacling/src/TClingClassInfo.cxx

```diff
@@ -634,6 +634,33 @@ ptrdiff_t TClingClassInfo::GetBaseOffset(TClingClassInfo* base, void* address, b
    return binfo.Offset(address, isDerivedObject);
 }
 
+std::vector<std::string> TClingClassInfo::GetUsingNamespaces()
+{
+   // Find and return all 'using' declarations of namespaces.
+   std::vector<std::string> res;
+
+   R__LOCKGUARD(gInterpreterMutex);
+
+   cling::Interpreter::PushTransactionRAII RAII(fInterp);
+   const clang::DeclContext *DC = cast<DeclContext>(fDecl);
```

Comment:
`const auto DC = dyn_cast<DeclContext>(fDecl));`? Else `fDecl` not being a `DeclContext` will crash.